static-analysis/autodispose-lint/src/main/kotlin/com/uber/autodispose/lint/AutoDisposeDetector.kt

```diff
@@ -76,11 +81,24 @@ class AutoDisposeDetector: Detector(), SourceCodeScanner {
     val evaluator = context.evaluator
 
     if (isReactiveType(evaluator, method) && isInScope(evaluator, node.getContainingUClass())) {
-      context.report(ISSUE, node, context.getLocation(node), "Always apply an AutoDispose " +
-          "scope before subscribing within defined scoped elements.")
+      context.report(ISSUE, node, context.getLocation(node), LINT_DESCRIPTION)
     }
   }
 
+  /**
+   * Checks if the calling method is in "scope" that can be handled by AutoDispose.
+   *
+   * If your subscribe/subscribeWith method is called in a scope
+   * that is recognized by AutoDispose, this returns true. This indicates that
+   * you're subscribing in a scope and therefore, you must handle the subscription.
+   * Default scopes include Android activities, fragments and custom classes that
+   * implement ScopeProvider.
+   *
+   * @see defaultScopes
```

Comment:
nit: this should go last

Reply:
06d7324829c34b6fcbb0ceb34b62597f45d2a7e8